### DISCUSSION TITLE
能出一个使用libuv来作为IO事件驱动库使用的Demo吗?

### DISCUSSION THREAD

**Discussion by tickmoo** [Tencent/tquic#321] [Q&A (问题和回答)]
相比于libev，目前libuv更常用，而且HarmonyOS Next 自带libuv 的库，省区重复交叉编译libev的麻烦了，使用更加通用的libuv作为demo不是更好吗？

**Comment by glime-ai** [comment]
使用libuv作为TQUIC的demo确实有其优势。libuv是一个跨平台的异步I/O库，广泛应用于各种项目中，包括Node.js。相比于libev，libuv的使用更为普遍，且HarmonyOS Next自带libuv库，这可以减少重复交叉编译libev的麻烦，提高开发效率。

然而，具体选择libev还是libuv作为demo库，可能还需要考虑以下几点：

1. **兼容性和依赖性**：确保TQUIC与libuv的兼容性，并且没有额外的依赖问题。
2. **性能**：评估libuv在TQUIC中的性能表现，确保其能够满足项目的需求。
3. **社区支持**：libuv拥有活跃的社区和丰富的文档，这对于开发和维护来说是一个优势。

如果您有兴趣，可以在TQUIC的GitHub仓库中提出这个建议，或者提交一个Pull Request来展示libuv的demo实现。

This comment was generated by [Glime](https://glimelab.ai/?utm_source=44520fce49ba43798485a628d6928e22&utm_medium=github).

**Comment by iyangsj** [comment]
后续版本会考虑添加基于libuv的示例程序，欢迎有兴趣的用户提交PR

**Reply by tickmoo** [reply]
很棒！
